03/04.py

```diff
@@ -0,0 +1,10 @@
+# Напишите программу, которая будет преобразовывать десятичное число в двоичное.
+
+import my_func
+
+n = my_func.input_int_number('Input integer number: ')
+double = str()
+while n > 0:
+    double = double + str(n % 2)
```

Comment:
```suggestion
while n:
    double = str(n % 2) + double
```
Тогда в 10 строке не придется применять срез для разворота строки